packages/application-shell-connectors/package.json

```diff
@@ -42,8 +42,8 @@
     "recompose": "0.26.0"
   },
   "devDependencies": {
-    "react": "16.6.1",
-    "react-apollo": "2.2.4"
+    "react": "16.6.3",
+    "react-apollo": "2.3.1"
```

Comment:
Server side rendering improvements and compatibility things with hooks.

Generally this would suffer from the same issue when not updating in the mc. Yarn here resolves to 2.3.1 while in the mc to 2.2.4 resulting in respective `node_modules` and likely double bundling.